### ISSUE TITLE
Move seed into experiment configuration file

### ISSUE THREAD

**Issue by lewfish** [azavea/raster-vision#14]
Currently the random seed is hardcoded. It makes more sense to have that as part of the RunOptions so it's possible to do multiple randomized runs.

**Comment by lewfish** [comment]
From this issue, it seems that currently there's no straightforward way to make experiments in Keras/Tensorflow reproducible. I doubt the random seed will have much of an effect on the final results, so I don't think this is critical right now.

**Comment by lewfish** [comment]
This issue is no longer applicable to the current and future versions of Raster Vision.
